### ISSUE TITLE
مشکل نصب کافه بازار بعد از اپ

### ISSUE THREAD

**Issue by mohsentta** [hojjatjafary/CafebazaarUnity#3]
سلام خسته نباشید. وقتی بعد از نصب اپی که پلاگین شما روشه کافه بازار نصب بشه پلاگین شما این ارور رو میده:
 AndroidJavaException: java.lang.SecurityException: Not allowed to bind to service Intent { act=ir.cafebazaar.pardakht.InAppBillingService.BIND pkg=com.farsitel.bazaar launchParam=MultiScreenLaunchParams { mDisplayId=0 mBaseDisplayId=0 mFlags=0 } }
09-23 19:22:09.266  8927  9268 E Unity   : java.lang.SecurityException: Not allowed to bind to service Intent { act=ir.cafebazaar.pardakht.InAppBillingService.BIND pkg=com.farsitel.bazaar launchParam=MultiScreenLaunchParams { mDisplayId=0 mBaseDisplayId=0 mFlags=0 } }

**Comment by hojjatjafary** [comment]
Are you sure that you added the permissions in your AndroidManifest file?

`<uses-permission android:name="com.farsitel.bazaar.permission.PAY_THROUGH_BAZAAR" />`

**Comment by mohsentta** [comment]
بله. پلاگین کاملا کار میکنه. فقط وقتی این ارور رو میده که کافه بعد از نصب اپ نصب بشه.

**Comment by hojjatjafary** [comment]
Try restarting the device after installing Bazaar app also check this issue:
https://github.com/cafebazaar/TrivialDrive/issues/5

**Comment by mohsentta** [comment]
are you kidding me? you should ask the permission through ActivityCompat.requestPermissions when i call Init() function. how I'm supposed to tell my user:
"hey bro restart your phone I'm getting permission denied error. 😅 "
please fix your plugin. we need this.

**Comment by hojjatjafary** [comment]
I just wanted to know that the issue remains after restarting or not but seems you know the solution then  please fixed it and send me a pull request, I'm very busy with my own job and have not enough time to fix and test the plugin.

**Comment by mohsentta** [comment]
i wrote a hotfix for it. it's inside the project i'm working on. i'll try to clone main project and pull the edits on it later. 
np we all have busy days.

**Comment by ParsaRt** [comment]
is it done yet ?

**Comment by oky2abbas** [comment]
+1
